sycl/CMakeLists.txt

```diff
@@ -243,6 +243,11 @@ add_custom_target(sycl-headers
           sycl-device-aspect-macros-header
           boost_mp11-headers)
 
+list(APPEND UR_HEADERS
```

Comment:
It looks like `UR_HEADERS` is originally defined in UR itself [here](https://github.com/oneapi-src/unified-runtime/blob/c34ea46276a92a1c125cdc28a79c31cd775b1308/CMakeLists.txt#L309) and these three files are UR files, should the fix instead be in UR itself rather than SYCL? This is fine as a temporary fix though

Reply:
The fix is really not to copy headers to sycl's build directory in the first place, but use `target_include_directories(PRIVATE`. The whole way we do dependencies isn't proper cmake, and I need to flense the whole thing. This is temporary

Reply:
I could change that variable name, though. Didn't realise it clashed

Reply:
i dont know enough about cmake anyway, i thought it was intentionally clashing :P

Reply:
Updated to use the delightfully quaint "UR_HEADERS_TO_COPY"